src/Data/DiGraph/FloydWarshall.hs

```diff
@@ -153,10 +166,10 @@ floydWarshallInternal
     -> Array U Ix2 (Double,Int)
 floydWarshallInternal a = foldl' go a [0..n-1]
   where
-    (n :. _) = size a
+    Sz (n :. _) = size a
```

Comment:
There is an incomplete pattern match here.

Reply:
I don't think it's incomplete. It think the pattern definition is just missing the `complete` pragma. Let me see if there is another backward compatible way to patch `n` out of the size...

Reply:
Well, it's also only a problem with `massiv-0.2.7`. Version 0.3 doesn't seem to have that problem, but `cabal2nix` fails on version 0.3 :-(